src/e-cherry-pick.js

```diff
@@ -155,7 +155,8 @@ program
           sha: commit.sha,
         });
 
-        const bugNumber = (/^Bug: (.+)$/m.exec(patch) || [])[1];
+        const bugNumber =
+          (/^Bug: (.+)$/m.exec(patch) || [])[1] || (/^Bug= ?chromium:(.+)$/m.exec(patch) || [])[1];
```

Comment:
Should also support:

```
BUG=chromium:1107593
```

```
Bug: chromium:1168451, chromium:1173961
```

```
Bug: chromium:1168451
Bug: chromium:1173961
```

```
Bug: chromium:1174013,chromium:1184272,chromium:1196203
```

```
Bug=chromium:1200169
```

```
Bug: 1176196, v8:11420
```

```
Bug: v8:11549
```

etc.

Reply:
Agreed, but not a blocker for merging this iteration.